solutions/swb-reference/SETUP_v2p1.md

```diff
@@ -142,49 +143,103 @@ Your environment should have four variables. Name the first one `API_URL` and th
 Import [SWBv2 Postman Collection](./SWBv2.postman_collection.json). Instructions for how to import a collection is [here](https://learning.postman.com/docs/getting-started/importing-and-exporting-data/#importing-data-into-postman)
 
 
-### Setup project configurations, environmentType, and environmentTypeConfig
+## Setup Account Resources
+### Onboard hosting account
+In **SWBv2 Official** Postman Collection under **hosting account** folder choose **Create Hosting Account** API to onboard a hosting account. 
+Remember to fill in the correct values for your account.
+Custom values that needed to be provided by you will be `<INSIDE THIS>`
+
+In the body tab set `envMgmtRoleArn` parameter to the `CFN_OUTPUT.EnvMgmtRoleArn` value from [Deploy to the Hosting Account step](#deploy-to-the-hosting-account).
 
-Since we have not built the API for Projects setup in DDB yet, you'll need to manually add these values to DDB. You can
-do so, by logging into the DDB page on the AWS console for your `Main account`. There should be a table there with the following
-name `swb-<stage>-<awsRegionShortName>`
+In the body tab set `hostingAccountHandlerRoleArn` parameter to the `CFN_OUTPUT.HostingAccountHandlerRoleArn` value from [Deploy to the Hosting Account step](#deploy-to-the-hosting-account).
 
-**Store `PROJ` resource in DDB**
 
-Remember to fill in the correct values for your account.
-Custom values that needed to be provided by you will be `<INSIDE THIS>`
+Send a **Create Hosting Account** request 
 
+POST `{{API_URL}}/aws-accounts`
 ```json
 {
-    "pk": "PROJ#cf3019e3-88d5-4a64-9025-26a177e36f59",
-    "sk": "PROJ#cf3019e3-88d5-4a64-9025-26a177e36f59",
-    "id": "cf3019e3-88d5-4a64-9025-26a177e36f59",
-    "accountId": "acc-123",
-    "indexId": "index-123",
-    "description": "Example project 1",
-    "owner": "abc",
-    "projectAdmins": [],
-    "resourceType": "project",
-    "name": "Project 1",
+    "name": "<Unique account name>",
+    "awsAccountId": "<Hosting Account 12 Digit ID>",
+    "envMgmtRoleArn": "<CFN_OUTPUT.EnvMgmtRoleArn>",
+    "hostingAccountHandlerRoleArn": "<CFN_OUTPUT.HostingAccountHandlerRoleArn>",
+    "externalId": "workbench"
+}
+```
+Once the request excecute successfully a response with the following format will be displayed
+```json
+{
+    "id": "acc-3123a751-2f5c-4b8e-bda7-e70e878257ty",
+    "name": "<Unique account name>",
+    "awsAccountId": "<Hosting Account 12 Digit ID>",
     "envMgmtRoleArn": "<CFN_OUTPUT.EnvMgmtRoleArn>",
     "hostingAccountHandlerRoleArn": "<CFN_OUTPUT.HostingAccountHandlerRoleArn>",
-    "encryptionKeyArn": "<CFN_OUTPUT.EncryptionKeyArn>",
-    "subnetId": "<CFN_OUTPUT.VpcSubnet>",
-    "vpcId": "<CFN_OUTPUT.VPC>",
     "externalId": "workbench",
```

Comment:
optional: `externalId` value need not be `workbench` always. You could replace it with <externalIdValue>

Reply:
fixed